cogment_verse/app.py

```diff
@@ -83,10 +83,10 @@ def __init__(self, cfg, work_dir=".cogment_verse"):
             if service_type not in [service_type.value for service_type in ServiceType]:
                 raise RuntimeError(f"Unknown service type [{service_type}]")
             if service_type == ServiceType.ORCHESTRATOR.value:
```

Comment:
Before it was considered a possibility to have more than one orchestrator?

Reply:
Yes (and it works fine) for the sake of simplicity I removed that for now (basically the idea is that if you have more than one orchestrator you'll have an "external" Cogment runtime (which is not yet supported).